jtlibrary/python/jtmodules/src/jtmodules/register_objects.handles.yaml

```diff
@@ -10,6 +10,11 @@ input:
         Binary or labeled mask image containing the objects
         (connected pixel components) that should be registered.
 
+    - name: relabel
+      type: Boolean
+      value: false
+      help: >
+        Should objects be relabeled on registration.
```

Comment:
Can we add a warning here that this shouldn't normally be done? Some users just seem to try out options at this is definitely an advanced feature